core/src/main/java/bio/terra/pearl/core/dao/study/StudyEnvironmentSurveyDao.java

```diff
@@ -32,15 +32,29 @@ public List<StudyEnvironmentSurvey> findAllByStudyEnvironmentId(UUID studyEnvId)
         return findAllByStudyEnvironmentId(studyEnvId, true);
     }
 
+    public List<StudyEnvironmentSurvey> findAllByStudyEnvironmentId(UUID studyEnvId, Boolean active) {
```

Comment:
Maybe this is my Scala bias showing, but I'm a fan of using `Optional` for these cases where we're knowingly supporting `null` parameters, since it hints in the signature that the empty/null case is expected and handled.

Reply:
I like that idea generally.  Here, to me at least, the use of Boolean instead of boolean is what communicates nulls are expected